### ISSUE TITLE
Feature request: decrease brightness on keypress

### ISSUE THREAD

**Issue by speller26** [WootingKb/wootility-issues#122]
**Is your feature request related to a problem? Please describe.**
Right now, analog RGB effects like AOE and Jelly will _increase_ brightness. starting from a minimum of **Idle Brightness** to a maximum of **Brightness**. If **Idle Brightness** is higher that **Brightness**, nothing happens on keypress.

**Describe the solution you'd like**
It would be nice for analog brightness to able to vary in either direction; increase if **Idle Brightness** < **Brightness** (like it does currently) and decrease if **Idle Brightness** > **Brightness** . In particular, I would like to have my keyboard at maximum brightness while idle, and a gradually dim, turning off completely when fully  depressed.

**Describe alternatives or workarounds you've considered**
No alternatives available right now, so I just regular AOE

**Additional context**
Basically, I'd like this to work:
![image](https://user-images.githubusercontent.com/15883075/145664362-d00a5ec1-f4ef-48c7-9e5c-81589a08cf0f.png)

### Additional information
**Platform information**
Windows 10

**Wootility and Firmware version**
Wootility v4.3.4-beta
Firmware v2.4.5

**Keyboard Type**
Wooting Two HE
